pkgs/applications/misc/yokadi/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  name = "yokadi-${version}";
+  version = "1.1.1";
+
+  src = fetchurl {
+    url = "https://yokadi.github.io/download/${name}.tar.bz2";
+    sha256 = "af201da66fd3a8435b2ccd932082ab9ff13f5f2e3d6cd3624f1ab81c577aaf17";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    dateutil
+    sqlalchemy
+    setproctitle
+    icalendar
+    pycrypto
+  ];
+
+  meta = with stdenv.lib; {
+    description = "A command line oriented, sqlite powered, todo-list";
+    homepage = https://yokadi.github.io/index.html;
+    license = licenses.gpl3;
+    maintainers = maintainers.nipav;
```

Comment:
This should be a list.

Reply:
So here I'm guessing that `maintainers` needs to be set to a list, such as this `maintainers = [ maintainers.nipav ];`?

Reply:
Correct.

Reply:
Ok, thank you. I'll fix everything else and push.